src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAppConfig.targets

```diff
@@ -0,0 +1,62 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.GenerateAppConfig.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="WriteAppConfig" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+  <UsingTask TaskName="SetGeneratedAppConfigMetadata" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
+
+  <PropertyGroup>
+    <_GenerateAppConfigIntermediateAppConfig>$(IntermediateOutputPath)$(TargetFileName).withSupportedRuntime.config</_GenerateAppConfigIntermediateAppConfig>
+  </PropertyGroup>
+
+  <Target Name="GenerateAppConfig"
+          Condition="'$(GenerateAppConfig)' != 'false' and '$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(HasRuntimeOutput)' == 'true'"
+          DependsOnTargets="_WriteAppConfig"
+          BeforeTargets="GenerateBindingRedirects">
```

Comment:
This means that if I set GenerateBindingRedirects to false, but GenerateAppConfig to true, that it won't work? I think in practice all SDK projects should generate binding redirects, but it feels a little clunky to tie them together.

Reply:
Fixed, renamed to GenerateSupportedRuntime and the task is called WriteAppConfigWithSupportedRuntime (since it will create one)

Reply:
Fixed, beforetargets `ResolveReferences`.